kinto_http/__init__.py

```diff
@@ -355,6 +361,26 @@ def delete_buckets(self, *, safe=True, if_match=None):
         resp, _ = self.session.request('delete', endpoint, headers=headers)
         return resp['data']
 
+    def get_history(self, *, bucket=None, **kwargs):
```

Comment:
I think it would be nice to raise an error if the `history` in not listed in the server capabilities